src/test/java/stepdefinitions/VerifySubmodulesSteps.java

```diff
@@ -0,0 +1,44 @@
+package stepdefinitions;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.WebDriver;
+import pages.LoginPage;
+import pages.SubmodulesPage;
+import utils.ConfigReader;
+import utils.DriverHelper;
+
+public class VerifySubmodulesSteps {
+
+    WebDriver driver = DriverHelper.getDriver();
+    LoginPage loginPage = new LoginPage(driver);
+    SubmodulesPage submodulesPage;
+
+    @Given("employee navigates to right webpage")
```

Comment:
Can you please explain please what this method does?

Reply:
Do you mean @Given? Navigates to url or you meant submodulesPage?

Reply:
I was referring to-->  ConfigReader.readProperty("techtorial_CRM_URL");